src/Settings/components/theme.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { useState } from "react";
+import { Text, Appearance } from "react-native";
+import { ListItem, BottomSheet, Avatar, Icon } from "@rneui/themed";
+import STRINGS from "../../common/localization";
+import styles from "../styles";
+import { setTheme, THEMES, toggleNightMode } from "../../common/actions";
+import { useSelector } from "react-redux";
+import colors from "../../common/colors";
+import constant from "../../common/constant";
+
+function ThemeComponent({ isNightMode, dispatch }) {
+  const [isVisible, toggleVisible] = useState(false);
+  const { theme } = useSelector((state) => state);
+
+  const toggleIsNightMode = (value) => {
```

Comment:
You can use better names for function like "handleThemeMode" which actually depicts the purpose of the function.